### DISCUSSION TITLE
Disable object minification (VueJS)

### DISCUSSION THREAD

**Discussion by xrnss** [babel/babel#12381] [Q&A] [answered]
I have my file like this:
```js
var eventBus = require("vue");

var obj= {};
obj.x = "y";
obj.events = new eventBus();

module.exports = obj;
```

However, when it is compiled, I get this
```js
// changed minified expressions so it's clear
var obj = { x: "y" }
obj.events = new eventBus()
```

Is there a way to it can just be the object then the line setting the value instead of in one line? I know it does not seem a big deal but if anyone could help I would be appreciated!

**Comment by nicolo-ribaudo** [comment, marked as answer]
By default Babel doesn't minify your code, you probably have enabled a minifier somewhere in your build pipeline.

**Reply by xrnss** [reply]
Do you know where this might be for Vue? I've disabled eslint if that changes anything

**Reply by nicolo-ribaudo** [reply]
Maybe https://github.com/vuejs/vue-cli/issues/4328#issuecomment-514250189 helps.

**Reply by xrnss** [reply]
Thanks for the help, do you know if Vue use Babel's minifier?

**Reply by nicolo-ribaudo** [reply]
I _think_ it uses `terser`, which is Webpack's default.
